README.md

```diff
@@ -18,8 +18,18 @@ Add spree_ams to your Spree store's Gemfile:
 gem 'spree_ams', :github => 'hhff/spree_ams', :branch => '2-3-alpha'
 ```
 
+Make sure `config.action_controller.asset_host` is set in your environment config if not running in production.
```

Comment:
you can keep using this PR, just add that method into the serializer (give it a test on your end - I just wrote it into github, haven't checked it)

still need to document in the README that `ActionController::Base.asset_host` should be set in any environment that doesn't produce an absolute URL to a paperclip asset